src/main/java/org/disq_bio/disq/impl/formats/bam/BamSink.java

```diff
@@ -60,14 +68,20 @@ public void save(
       JavaRDD<SAMRecord> reads,
       String path,
       String referenceSourcePath,
-      String tempPartsDirectory)
+      String tempPartsDirectory,
+      long sbiIndexGranularity)
       throws IOException {
 
     Broadcast<SAMFileHeader> headerBroadcast = jsc.broadcast(header);
+    boolean writeSbiFile = header.getSortOrder() == SAMFileHeader.SortOrder.coordinate;
```

Comment:
Can't we write SBI files for non-coordinate sorted bams as well?  I thought we could generate one even on unmapped bams.

Reply:
Good point. Fixed